MANIFEST.in

```diff
@@ -1,9 +1,10 @@
 include *.txt
 recursive-include wof/flask/templates *
-recursive-include wof/examples/flask/odm2/timeseries *
+recursive-include wof/examples *
```

Comment:
@ocefpaf Doesn't `wof/examples` automatically get added since it is a python module? Just trying to understand why it needs to be explicitly defined. Thanks.